StreamChatClient_v3/Models/ChannelModel.swift

```diff
@@ -8,9 +8,6 @@ import Foundation
 public struct ChannelModel<ExtraData: ExtraDataTypes> {
     // MARK: - Public
     
-    /// A channel type.
-    public let type: ChannelType
-    
     /// A channel type + id.
     public let id: ChannelId
```

Comment:
I know it sounds weird, but other SDK's use `cid` for channel id = type + id and the `id` itself is used in conversations as just id without type. So, it could be misunderstanding is that the right `cid` or just `id`.
So, I think we should rename it to `cid`.

Related to this: `id.idWithoutType.hasPrefix(...)` looks weird. It could be `cid.id.hasPrefix(...)`

Reply:
OK, fair enough 👍

Reply:
fixed in 2b15da631bf8292589f95ee7f31433b3c8275ad7